react/features/invite/actions.js

```diff
@@ -1,16 +1,50 @@
-/* globals APP */
+/* globals $, APP, config */
```

Comment:
If I remember correctly @lyubomir prefers that we use the declare var $: Function; syntax for global variables.